apps/chat/README.md

```diff
@@ -24,10 +39,11 @@ The easiest way to get started with messaging in the Amazon Chime SDK is to depl
 
 After the deployment is complete, the **Outputs** tab of the AWS CloudFormation console includes your resource information. You will update the configuration file with these values in the next section.
 
-## Running the Amazon Chime SDK Chat Demo
+## Running the chat demo app locally
 
 1. Navigate to the root directory of the Amazon Chime SDK chat demo `apps/chat`.
-1. Open `src/Config.js` and update the configuration with the values from the previous section.
+1. If you have deployed the CloudFormation stack manually in the previous steps, open `src/Config.js` and update the configuration with 
```

Comment:
nit: If you modified the above title to `Using the AWS CloudFormation console`, you can change this instruction to the following:

> If you have used the AWS CloudFormation console to deploy the stack in the previous steps, open ...

Reply:
updated